templates/cisco_ios_show_object-group.textfsm

```diff
@@ -0,0 +1,31 @@
+Value Required,Filldown OGR_TYPE (Service|Network)
+Value Required,Filldown OGR_NAME (\S+)
+Value DESCRIPTION (.+)
+Value NESTED_OGR (\S+)
+Value ANY (any)
+Value HOST (\d+\.\d+\.\d+\.\d+)
+Value HOST_RANGE_START (\d+\.\d+\.\d+\.\d+)
+Value HOST_RANGE_END (\d+\.\d+\.\d+\.\d+)
+Value NETWORK (\d+\.\d+\.\d+\.\d+)
+Value NETMASK (\d+\.\d+\.\d+\.\d+)
+Value PROTOCOL (\S+)
+Value PORT_MATCH (eq|neq|range|lt|gt)
+Value PORT ((?<!range\s)\S+)
+Value PORT_RANGE_START ((?<=range\s)\S+)
+Value PORT_RANGE_END (\S+)
+Value ICMP_TYPE (echo|echo-reply|administratively-prohibited|unreachable|port-unreachable|redirect|router-advertisement|router-solicitation|packet-too-big|time-exceeded|ttl-exceeded|parameter-problem)
+
+Start
+  ^(Service|Network) -> Continue.Clearall
```

Comment:
We should be able to capturing everything within single entries.

Look into: https://github.com/networktocode/ntc-templates/blob/master/templates/cisco_ios_show_ip_interface.textfsm for how to accomplish this.

Reply:
Along this line, you will want to remove all the `-> Record` on child lines.

Reply:
Let me discuss internally as this is quite a different case, and need to make sure we're aligned on how we should capture the data.

Reply:
Ignore this comment and I will resolve any other feedback that doesn't need to be addressed.